tests/test_rds2/test_rds2.py

```diff
@@ -31,6 +30,32 @@ def test_create_database():
 
 @disable_on_py3()
 @mock_rds2
+def test_create_database_triggers_event():
+    def on_database_created(_event_id, database):
+        database.master_username = database.master_username[::-1]  # reverse
+        database.port = 4567
+        database.address_gen = lambda instance_id, region: "{0}.{1}.somewhere-else.com".format(instance_id, region)
+
+    default_publisher.subscribe(on_database_created, EventTypes.RDS2_DATABASE_CREATED)
+
+    conn = boto3.client('rds', region_name='us-west-2')
+    instance = conn.create_db_instance(DBInstanceIdentifier='db-master-1',
+                                       AllocatedStorage=10,
+                                       Engine='postgres',
+                                       DBInstanceClass='db.m1.small',
+                                       MasterUsername='root',
+                                       MasterUserPassword='hunter2',
+                                       Port=1234,
+                                       DBSecurityGroups=["my_sg"])['DBInstance']
+
+    instance['MasterUsername'].should.equal("toor")
+    instance['Endpoint']['Port'].should.equal(4567)
+    instance['Endpoint']['Address'].should.equal("db-master-1.us-west-2.somewhere-else.com")
+    default_publisher.reset()
+
+
+@disable_on_py3()
```

Comment:
FYI, all these tests seem to work with python 3 (at least python 3.5). I've kept them like that since I don't know the reason why they've been disabled.